packages/integrations/src/prowlarr/prowlarr-integration.ts

```diff
@@ -0,0 +1,83 @@
+import { Integration } from "../base/integration";
+import { IntegrationTestConnectionError } from "../base/test-connection-error";
+import type { Indexer } from "../interfaces/indexer-manager/indexer";
+import { indexerResponseSchema, statusResponseSchema } from "./prowlarr-types";
+
+export class ProwlarrIntegration extends Integration {
+  public async getIndexersAsync(): Promise<Indexer[]> {
+    const apiKey = super.getSecretValue("apiKey");
+
+    const indexerResponse = await fetch(`${this.integration.url}/api/v1/indexer?apikey=${apiKey}`);
+    if (!indexerResponse.ok) {
+      throw new Error(
+        `Failed to fetch indexers for ${this.integration.name} (${this.integration.id}): ${indexerResponse.statusText}`,
+      );
+    }
+
+    const statusResponse = await fetch(`${this.integration.url}/api/v1/indexerstatus?apikey=${apiKey}`);
```

Comment:
Does this one only return disabled indexers?

Reply:
Yes